docs/reference/system-requirements.rst

```diff
@@ -84,6 +84,7 @@ AMD ROCm software supports the following Linux distributions.
     :header: "Operating system", "Kernel", "Support"
     :escape: \
 
+    "Ubuntu 24.04.1", "6.8 [GA]", "✅"
     "Ubuntu 24.04", "6.8 [GA]", "✅"
     "Ubuntu 22.04.5", "5.15 [GA], 6.8 [HWE]", "✅ [#Ubuntu22045]_"
```

Comment:
```suggestion
    "Ubuntu 22.04.5", "5.15 [GA], 6.8 [HWE]", "✅"
```

Reply:
Also, can you remove the footnote at the bottom of the page for: [#Ubuntu22045]?  Line ~128.
Looks like I forgot to remove this since it's no longer a preview OS.